docs/BasicCommunication/OnAppRegistered/index.md

```diff
@@ -1,13 +1,10 @@
 ## OnAppRegistered
 
-Type
-: Notification
+Type: Notification
 
-Sender
-: SDL
+Sender: SDL
 
-Purpose
-: Update the HMI's list of registered applications and resume the Audio Source
+Purpose: Update the HMI's list of registered applications and resume the Audio Source
```

Comment:
@DrachenkoAnastasiia Alright, it looks like I was wrong about the fact that the keys should be on the same line as the value (although I have no idea why it makes a difference). However, it does appear as if a newline between key:value pairs is necessary. I will fix the formatting on these.